crates/e2e/tests/e2e/protocol_fee.rs

```diff
@@ -326,36 +325,6 @@ async fn combined_protocol_fees(web3: Web3) {
         .saturating_sub(limit_surplus_order.data.buy_amount);
     // see `limit_surplus_policy.factor`, which is 0.3
     assert!(limit_executed_surplus_fee_in_buy_token >= limit_quote_diff * 3 / 10);
-
-    let [market_order_token_balance, limit_order_token_balance, partner_fee_order_token_balance] =
-        futures::future::try_join_all(
-            [
-                &market_order_token,
-                &limit_order_token,
-                &partner_fee_order_token,
-            ]
-            .map(|token| {
-                token
-                    .balance_of(onchain.contracts().gp_settlement.address())
-                    .call()
-            }),
-        )
-        .await
-        .unwrap()
-        .try_into()
-        .expect("Expected exactly four elements");
-    assert_approximately_eq!(
-        market_executed_surplus_fee_in_buy_token,
-        market_order_token_balance
-    );
-    assert_approximately_eq!(
-        limit_executed_surplus_fee_in_buy_token,
-        limit_order_token_balance
-    );
-    assert_approximately_eq!(
-        partner_fee_executed_surplus_fee_in_buy_token,
-        partner_fee_order_token_balance
-    );
```

Comment:
Executed surplus fee cant longer be used to check the amount of fee kept in the settlement contract.